{"data_version": "4.0","references": {"reference_data": [{"url": "https://github.com/rohe/pysaml2/issues/451"}]},"description": {"description_data": [{"lang": "eng","value": "pysaml2 version 4.4.0 and older accept any password when run with python optimizations enabled. This allows attackers to log in as any user without knowing their password."}]},"data_type": "CVE","affects": {"vendor": {"vendor_data": [{"product": {"product_data": [{"version": {"version_data": [{"version_value": "4.4.0 and older"}]},"product_name": "pysaml2"}]},"vendor_name": "pysaml2"}]}},"CVE_data_meta": {"DATE_ASSIGNED": "2017-12-29","ID": "CVE-2017-1000433","ASSIGNER": "kurt@seifried.org","REQUESTER": "predrag.gruevski@gmail.com"},"data_format": "MITRE","problemtype": {"problemtype_data": [{"description": [{"lang": "eng","value": "Incorrect Access Control"}]}]}}